torchtitan/experiments/flux/modules/HFEmbedder.py

```diff
@@ -0,0 +1,45 @@
+from torch import nn, Tensor
+from transformers import CLIPTextModel, CLIPTokenizer, T5EncoderModel, T5Tokenizer
+
+
+class HFEmbedder(nn.Module):
```

Comment:
noob question, what does this HF stand for? hugging face?

Reply:
The name was adopted from Flux repo, my understanding is HF=hugging_face. Then embedder is downloaded from hugging face :)